simple-podcasting.php

```diff
@@ -120,7 +120,7 @@ function podcasting_edit_term_enqueues( $hook_suffix ) {
 		);
 	}
 
-	if ( 'admin_page_simple-podcasting-onboarding' === $hook_suffix ) {
+	if ( in_array( $hook_suffix, $screens, true ) ) {
```

Comment:
`admin_page_simple-podcasting-onboarding` should be added to the `$screens` array.
```suggestion
        $screens[] = 'admin_page_simple-podcasting-onboarding';
	if ( in_array( $hook_suffix, $screens, true ) ) {
```